pkgs/development/python-modules/tilequant/default.nix

```diff
@@ -0,0 +1,34 @@
+{ lib, buildPythonPackage, fetchFromGitHub, GitPython, click, ordered-set, pillow, sortedcollections }:
+
+let
+  aikku93-tilequant = fetchFromGitHub {
+    owner = "SkyTemple";
+    repo = "aikku93-tilequant";
+    rev = "6604e0906edff384b6c8d4cde03e6601731f66fd";
+    sha256 = "0w19h3n2i0xriqsy0b0rifjgbv4hqd7gl78fw0cappkrdykij5r1";
+  };
+in
+buildPythonPackage rec {
+  pname = "tilequant";
+  version = "0.4.0.post0";
+
+  src = fetchFromGitHub {
+    owner = "SkyTemple";
+    repo = pname;
+    rev = version;
+    sha256 = "189af203iay3inj1bbgm3hh1fshn879bcm28ypbvfp27fy7j5b25";
+  };
+
+  patchPhase = "cp -R --no-preserve=mode ${aikku93-tilequant} __aikku93_tilequant";
```

Comment:
```suggestion
  postPhase = ''
    cp -r --no-preserve=mode ${aikku93-tilequant} __aikku93_tilequant
  '';
```

Reply:
What is `postPhase`? I don't see that used anywhere in nixpkgs, and changing `patchPhase` with `postPhase` causes the code to not be executed.

Reply:
I think he meant `postPatch`
Instead of overriding a phase directly it's more common to include customization in a post hook